web/lib/mobx/store-init.tsx

```diff
@@ -58,12 +58,24 @@ const MobxStoreInit = observer(() => {
    */
   useEffect(() => {
     if (workspaceSlug) workspaceStore.setWorkspaceSlug(workspaceSlug.toString());
+
     if (projectId) projectStore.setProjectId(projectId.toString());
+    else projectStore.setProjectId(null);
+
     if (cycleId) cycleStore.setCycleId(cycleId.toString());
+    else cycleStore.setCycleId(null);
```

Comment:
```suggestion
    cycleStore.setCycleId(cycleId?.toString() ?? null);
```

Reply:
and repeat for all please. this would reduce the repetition. `||` or `??` whatever the standard here is

Reply:
also, if its possible to use undefined instead of null to do the same, it wouldnt even require `?? null` and make the code alot more elegant